sdk/typespec/typespec_client_core/Cargo.toml

```
@@ -49,6 +51,7 @@ default = [
   "reqwest_rustls",
   "tokio_sleep",
 ]
+derive = ["typespec_derive"]
```

```suggestion
derive = ["dep:typespec_derive"]
```

Just to make sure no implicit `typespec_derive` feature is created, or I may be misunderstanding the docs about this. Still, I've been trying to be consistent to avoid confusion.